### PR TITLE
Add compass chart

### PR DESCRIPTION
**Pull request by pkosiec**
**Description**

Changes proposed in this pull request:

- Add compass chart

**Related issue(s)**
See #4242

### CODE REVIEW

File: installation/scripts/minikube.sh [hunk 1]
```diff
@@ -219,7 +219,7 @@ function start() {
...
+    addDevDomainsToEtcHosts "apiserver.${MINIKUBE_DOMAIN} console.${MINIKUBE_DOMAIN} catalog.${MINIKUBE_DOMAIN} instances.${MINIKUBE_DOMAIN} brokers.${MINIKUBE_DOMAIN} dex.${MINIKUBE_DOMAIN} docs.${MINIKUBE_DOMAIN} add-ons.${MINIKUBE_DOMAIN} lambdas-ui.${MINIKUBE_DOMAIN} console-backend.${MINIKUBE_DOMAIN} minio.${MINIKUBE_DOMAIN} jaeger.${MINIKUBE_DOMAIN} grafana.${MINIKUBE_DOMAIN} log-ui.${MINIKUBE_DOMAIN} loki.${MINIKUBE_DOMAIN} configurations-generator.${MINIKUBE_DOMAIN} gateway.${MINIKUBE_DOMAIN} connector-service.${MINIKUBE_DOMAIN} oauth2.${MINIKUBE_DOMAIN} oauth2-admin.${MINIKUBE_DOMAIN} oauth2-login-consent.${MINIKUBE_DOMAIN} oathkeeper-proxy.${MINIKUBE_DOMAIN} oathkeeper-api-server.${MINIKUBE_DOMAIN} kiali.${MINIKUBE_DOMAIN} compass-gateway.${MINIKUBE_DOMAIN}"
```
Comment (crabtree) [inline installation/scripts/minikube.sh]: Could you please add also compass-gateway in minikube.ps1

Comment (pkosiec) [reply installation/scripts/minikube.sh]: Thanks, done.

File: resources/compass/charts/gateway/templates/virtualservice.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  labels:
+    app: {{ template "name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+spec:
+  hosts:
+    - '{{ .Values.virtualService.host }}.{{ .Values.global.ingress.domainName }}'
+  gateways:
+    - {{ .Values.global.istio.gateway.name }}
```
Comment (crabtree) [inline resources/compass/charts/gateway/templates/virtualservice.yaml]: You should use existing virtual service from kyma-system here. As far as I know you cannot register two gateways in istio for the same domain (in this case for example *.kyma.local).

File: resources/compass/charts/gateway/templates/gateway.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+apiVersion: networking.istio.io/v1alpha3
```
Comment (crabtree) [inline resources/compass/charts/gateway/templates/gateway.yaml]: Please see my comment in file `resources/compass/charts/gateway/templates/virtualservice.yaml` line 15.

File: resources/compass/values.yaml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    connector:
+      dir: pr/
+      version: "PR-59"
+    director:
+      dir: pr/
+      version: "PR-79"
+    gateway:
+      dir: pr/
+      version: "PR-59"
+    healthchecker:
```
Comment (PK85) [inline resources/compass/values.yaml]: If dummy container, we should remove that from chart hmm?

Comment (pkosiec) [reply resources/compass/values.yaml]: Let's keep it - we would need to add it again soon 🙂

**Pull request merged**